### DISCUSSION TITLE
Mosaic LSM gives instantaneous soil moisture (SoilMoist_inst) or time average soil moisture (SoilMoist_tavg)?

### DISCUSSION THREAD

**Discussion by Gayathridevi123** [NASA-LIS/LISF#1054] [Q&A] [answered]
Hi LIS Community,

I would like to ask about the output of Mosaic LSM. I generated daily output HIST files (**Surface model output interval: "1da"**) having variables like EF (evaporative fraction), Rainf (Rainfall rate), SoilMoist (Average layer soil moisture), etc.

But in the output files, for soil moisture, the variable name is **instantaneous soil moisture (SoilMoist_inst)** not SoilMoist_tavg. Like that for some other variables also showing as instantaneous (SoilTemp_inst, Albedo_inst, RootMoist_inst, etc).  So is it giving daily instantaneous soil moisture from 00UTC or daily average soil moisture?

One sample HIST file is attached. 
[LIS_HIST.zip](https://github.com/NASA-LIS/LISF/files/8471290/LIS_HIST.zip

Thank you,
Gayathri

**Comment by dmocko** [comment, marked as answer]
Variable names in the netCDF files that end with "_inst" are instantaneous values, valid at the exact time of the output file.

Variable names in the netCDF files that end with "_tavg" are backward-averaged over the output interval.  So, if you have a "1da" output interval, and the time of the file is listed as 00Z02Jun2020, then the field will represent the average over 00Z01Jun2020 to 00Z02Jun2020.

Users can choose to make each variable either instantaneous or time-averaged (or output both) by changing the MODEL_OUTPUT_LIST.TBL file, as specified in your lis.config:
    https://github.com/NASA-LIS/LISF/blob/master/lis/configs/MODEL_OUTPUT_LIST.TBL.adoc

The fourth column (the first column after the "Sign Convention" column) is what you need to change.

**Reply by Gayathridevi123** [reply]
Thank you David for your reply.
